app/controllers/v0/map_services_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module V0
+  class MapServicesController < ApplicationController
+    skip_before_action :authenticate
+    before_action :authenticate_service_account
+
+    # POST /v0/map_services/token
+    def token
+      result = MAP::SecurityToken::Service.new.token(application: params[:application], icn: params[:icn])
```

Comment:
❓ I should probably get the value for `application` from the service account associated with the token instead of from a param. What do y'all think?

Reply:
I ended up adding a method to load the service account config so we could pull the `service_account_id` for use here instead of a param.